app/helpers/lessons_helper.rb

```diff
@@ -3,6 +3,10 @@ def github_edit_url(lesson)
     github_link("curriculum/edit/main#{lesson.github_path}")
   end
 
+  def github_commits_url(lesson)
```

Comment:
Nit: Can we add a unit test for this method please? It will almost identical to [the github_edit_url unit test](https://github.com/TheOdinProject/theodinproject/blob/main/spec/helpers/lessons_helper_spec.rb#L4).